website/content/en/preview/troubleshooting.md

```diff
@@ -33,17 +33,34 @@ Reasons that a node can fail to join the cluster include:
 - Security Groups
 - Networking
 
-The easiest way to start debugging is to connect to the instance
-```sh
+The easiest way to start debugging is to connect to the instance and get the Kubelet logs.  For an AL2 based node:
```

Comment:
Consider pointing to the installation of the SSM manager as a DS [here](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/install-ssm-agent-on-amazon-eks-worker-nodes-by-using-kubernetes-daemonset.html)

Reply:
might be confusing to post that link since AL2 has the SSM Agent already installed by default

Reply:
Ah, didn't know that, good to know :)